src/PreCICE.jl

```diff
@@ -1332,5 +1340,149 @@ function actionReadIterationCheckpoint()
     return unsafe_string(msgCstring)
 end
 
+@doc """
+
+        isGradientDataRequired(dataID::Integer)
```

Comment:
Only one tab indentation, also you might want to add `::Bool` at the end for the user to know what is returned